src/avfoundation.cs

```diff
@@ -10991,13 +10991,15 @@ interface AVPlayerLooper
 		[DesignatedInitializer]
 		IntPtr Constructor (AVQueuePlayer player, AVPlayerItem itemToLoop, CMTimeRange loopRange);
 
+#if !XAMCORE_4_0 // This API got introduced in Xcode 8.0 binding but is not currently present nor in Xcode 8.3 or Xcode 9.0 needs research
+		[Internal]
+		[Export ("disableLooping")]
+		void _DisableLooping ();
+#else
 		[Export ("disableLooping")]
 		void DisableLooping ();
```

Comment:
If I follow the logic of this patch: "API got introduced in Xcode 8.0 binding but is not currently present" then I don't see why `DisableLooping` should remain in the else clause. Nothing tells use that if we ever go `XAMCORE_4_0` this API will be there (and working).